jekyll/_cci2/server/Installation/migrate-from-server-3-to-server-4.adoc

```diff
@@ -171,22 +184,14 @@ The following Kubernetes service object is added:
 
 * circleci-proxy (LoadBalancer)
 
-[#step-8-validate-your-migration-to-server-4]
-=== Step 8 - Validate your migration to server v4.x
-
-Re-run https://support.circleci.com/hc/en-us/articles/360011235534-Using-realitycheck-to-validate-your-CircleCI-installation[realitycheck] on your new server 4.x environment by pushing a fresh commit.
-
-[#step-9-execute-cleanup-step]
-=== Step 9 - Execute cleanup step
-Execute the following cleanup step to remove the `kots` annotation and pod:
+[#step-8-execute-nomad-terraform]
+=== Step 8 - Execute Nomad Terraform
+Execute the link:https://github.com/CircleCI-Public/server-terraform[Nomad Terraform] to re-create nomad client where `server_endpoint` value is set to be `<domain>:4647`. You can follow the steps mentioned link:https://circleci.com/docs/2.0/server-4-install-build-services#nomad-clients[here].
```

Comment:
I don't know what "execute the link" means. What are the specific things someone has to do here?

Reply:
@circlery It is - Execute the "Nomad Terraform" , `link` is a markdown to add the hyperlink